index.js

```diff
@@ -109,23 +126,29 @@ Fireproof._handleError = function(onComplete) {
 /**
  * Tell Fireproof to use a given promise library from now on.
  * @method Fireproof.bless
- * @param {Q} Q a Q-style promise constructor with at least defer().
- * @throws if you don't provide a valid Deferred-style promise library.
+ * @param {Q} Q a Q-style promise constructor with an optional defer().
+ * @throws if you don't provide a valid promise library.
  */
 Fireproof.bless = function(newQ) {
 
-  if (newQ === undefined || newQ === null || typeof(newQ.defer) !== 'function') {
-    throw new Error('You tried to give Fireproof an invalid Q library!');
-  }
-
-  var deferred = newQ.defer();
+  function assert(value) {if (!value) throw null}
```

Comment:
please break out the indentation on this function, and move the exception from line 151 to here rather than throwing `null` - that way we can save the extra `throw`.